src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+.product-card {
+  box-sizing: border-box;
```

Comment:
will be better to set this property by default for all elements using * selector

Reply:
agree